docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -1,23 +1,28 @@
 [id='optimizer-download-proc']
 = Download {PLANNER}
 
-{PLANNER} is production ready.
-The API is stable but backward incompatible changes can occur.
+{PLANNER} is production ready, and can be used to optimize business solutions in {CENTRAL}, in your preferred IDE, or on OpenShift.
 
 .Prerequisite
 The {PRODUCT} has been downloaded and installed from the https://access.redhat.com[Red Hat Customer Portal]. See Installing Decision Manager for more information about installing {PRODUCT}.
 //@doc-link:Add link to Installation assembly above. 
 
 .Procedure
-. Download the {PLANNER} file from the https://access.redhat.com[Red Hat Customer Portal].
-. Unzip the {PLANNER} files.
+. Download and install {PRODUCT} from the https://access.redhat.com[Red Hat Customer Portal]. See the _Getting started with decision services_ for more information about installing {PRODUCT}.
+. Download the {PLANNER} from the https://access.redhat.com[Red Hat Customer Portal].
+. Extract the {PLANNER} files to access the complete set of {PLANNER} engine JAR files. 
+. xref:optimizer-running-the-examples-proc[Run the examples].
+
+
+{PLANNER} is production ready.
+The API is stable, however, backward incompatible changes can occur.
 
 
 ////
 [float]
 == Upgrading from previous versions
 
-With the recipe called https://www.optaplanner.org/download/upgradeRecipe/[UpgradeFromPreviousVersionRecipe.txt], you can easily upgrade to a newer version and quickly deal with any backwards incompatible changes.
+You can upgrade to a newer version and quickly deal with any backwards incompatibility changes using the https://www.optaplanner.org/download/upgradeRecipe/[UpgradeFromPreviousVersionRecipe.txt].
 
 This recipe file is included in every release.
 ////
```

Comment:
Looks like this whole block is commented out. I can't see it in the PDF. If it shouldn't be in the output, we should remove it instead commenting it out. Or perhaps move it to the migration guide.

Reply:
As above.